docs/user-guides/notifications/catalog/templates.md

```diff
@@ -0,0 +1,101 @@
+# Triggers Catalog
+
+This document provides a catalog of recommended [templates] for common use cases
+in Config Sync. This catalog uses email as an example, but these can be modified
+to work for other services as well. See the [services] reference doc for the
+template configuration API of each service.
+
+The set of [built-in functions](./functions.md) can be invoked from templates.
+
+## reconciler-created
+
+```yaml
+  template.reconciler-created: |
+    message: |
+      {{.sync.kind}} {{.sync.metadata.namespace}}/{{.sync.metadata.name}} has been created.
+    email:
+      subject: |
+        A new {{.sync.kind}} has been created.
+```
+
+## reconciler-deleted
+
+```yaml
+  template.reconciler-deleted: |
+    message: |
+      {{.sync.kind}} {{.sync.metadata.namespace}}/{{.sync.metadata.name}} has been deleted.
+    email:
+      subject: |
+        A {{.sync.kind}} has been deleted.
+```
+
+## reconciler-stalled
+
+```yaml
+  template.reconciler-stalled: |
+    message: |
+      {{.sync.kind}} {{.sync.metadata.namespace}}/{{.sync.metadata.name}} is Stalled.
+      
+      {{range $index, $e := call .utils.ConfigSyncErrors}}{{js $e.ErrorMessage}}\n\n{{end}}
```

Comment:
The stalled error should be included in the Stalled condition, instead of ConfigSyncErrors.

Reply:
I don't think it is necessary add `conditions` to `ConfigSyncErrors`.
For STALLED and RECONCILING conditions, the error is from its condition, not `ConfigSyncErrors`.
For SYNCING condition, the error is from `ConfigSyncErrors`, not the condition.